### ISSUE TITLE
bug(Selector/option): Option status inside a selector not read when "selected"

### ISSUE THREAD

**Issue by emaestromartinez** [angular/components#29040]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When using the screenreader to read the selected/unselect, NVDA doesn't announce the 'Selected' state for selected list item when user navigates on list items.
NVDA only announces list item and its position in the list.
It does announce when they are selected.

Expected behavior:
It should also announce "Selected" for selected items, not just the name and position on the list.

### Reproduction

Use angular material Select component as an example.
https://v7.material.angular.io/components/select/overview#multiple-selection
Navigate through the list using keyboard navigation. Have some options selected and others unselected. Use NVDA to read the state of the options.

### Expected Behavior

It should also announce "Selected" for selected items, not just the name and position on the list.

### Actual Behavior

When using the NVDA screenreader to read the selected/unselect, NVDA doesn't announce the 'Selected' state for selected list item when user navigates on list items.
NVDA only announces list item and its position in the list.
It does announce when they are selected.

### Environment

Angular 17~, Angular Material 17~
- Browser(s): Chrome last table version
- Operating System: Windows 10 Enterprise. Last Stable version.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
